package.json

```diff
@@ -97,7 +97,7 @@
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^13.3.7",
+    "@angular-devkit/build-angular": "^13.3.9",
```

Comment:
Minor upgrades to test tools. These are all working. I feel it's work keeping the updates happening.